src/openrct2/Context.cpp

```diff
@@ -448,10 +449,10 @@ namespace OpenRCT2
             util_srand((uint32)time(nullptr));
             input_reset_place_obj_modifier();
             viewport_init_all();
-            game_init_all(150);
+            _gameState->InitAll(150);
```

Comment:
You are calling a member of _gameState before creating it (on the next line).

Reply:
Thanks, that sorts it. Not sure how I missed it...